en/patterns/hierarchical-grid.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Hierarchical Grid - Design System Pattern
+_description: The Hierarchical Grid Pattern Symbol represents a parent-level Grid with a nested child-Grid inside.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Hierarchical Grid
+
+Use the Hierarchical Grid Pattern symbol as a starting point when designing a complex Hierarchical Grid with nested child Grids and some of its features. It provides a parent Grid and nested child Grid with a selected cell and visual representation of the connection between the parent and child grids. On Hierarchical Grid can be applied all the Grid functionality, provided in the Components Library such as:
+
+- Sorting
+- Filtering
+- Summaries
+- Paging
+- Editing
+- Column Moving
+- Column Resizing
+- Column Hiding
+- Display Density
+- Excel Style Filtering
+- GroupBy
+- Pinning
+- Row Selection
+- Row Editing
+- Active Cell
+- Export
+- Toolbar
+
+All Grid features are applicable to each child and parent Grid.
+
+<img class="responsive-img" src="../images/hierarchical_grid.png" srcset="../images/hierarchical_grid@2x.png 2x" />
```

Comment:
Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.